### ISSUE TITLE
Add list of ICF delegations to this repo

### ISSUE THREAD

**Issue by fkbenjamin** [interchainio/delegation#33]
I think it would be very helpful if this repo would - for further transparency -  contain a list of all delegations made by the ICF and the amount of the delegation. This would especially help people who are not able to run the respective [go file](https://raw.githubusercontent.com/interchainio/delegation/master/delegations/2-match-small/main.go) on their own. It also helps to keep track of changes in delegations.

I would be happy to create the list as a PR, I would just need a list of all ICF addresses that are currently delegating as part of this repo.

**Comment by ebuchman** [comment]
Yes a concise a summary of all delegations made would be super valuable. There's only one address currently making delegations: cosmos1unc788q8md2jymsns24eyhua58palg5kc7cstv

Presumably this would be distinct from just running `gaiacli query staking delegations cosmos1unc788q8md2jymsns24eyhua58palg5kc7cstv`, which only shows current delegations.

If you have a script that can generate that list and want to provide it with the list, that'd be great, though we'll want to keep things somewhat organized so it's clear which round each delegation was associated with. 

Thanks!

**Comment by valuead** [comment]
Guys, let me ask you a question.  All 100 people invested in infrastructure, build systems, monitor and trying to make cosmos run smooth, don't sleep, many waited long time and believe in cosmos success.
Someone started early someone starting just now. Speaking it for my self, I just started a validator and from 2017 I participated at gaia testnets from early days. I build a website and talk to communities around the word and crypto forums about cosmos. Just in few days I'll become a validator above 100 line. Yes, I did participate is ICO and I'm asking ICF to review this proposal on cosmos governance thru VOTINGS.
There is a group of people who are not finding this fair, clean and transparent. This distribution should be done across all validators. What about 25% new validators that coming on board on July 17th
This should be resolved thru governance/proposals by cosmos community.
Kindly asking to review.
